### PR TITLE
chore: add DpUploadFiles to Storybook.

### PR DESCRIPTION
**Pull request by ahmad-demos**
- add DpUploadFiles to Storybook.

### CODE REVIEW

File: src/components/core/DpUpload/DpUploadFiles.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+<Meta
+    title="Components/UploadFiles"
+    component={DpUploadFiles}
+/>
+
+# UploadFiles
+
+The UploadFiles component can be upload a single file or multiple files.
+
+The `allowedFileTypes` Prop allows us to choose the type of files that we want to upload.
```
Comment (spiess-demos) [inline src/components/core/DpUpload/DpUploadFiles.stories.mdx:13]: I see that props within DpUploadFiles.vue are not documented equally helpful. Apart from the need to document all props, i'd opt for examples with...

- only allow zip uploads
- max file size is limited
- @file-remove and @upload-success is used

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: src/components/core/DpUpload/DpUploadFiles.stories.mdx (hunk 1 above)
Comment (ahmad-demos) [reply src/components/core/DpUpload/DpUploadFiles.stories.mdx]: Done in https://github.com/demos-europe/demosplan-ui/pull/518/commits/2999afd2b3761940bacc0dabf0dee335914801bc
Is this okay?

File: src/components/core/DpUpload/DpUploadFiles.vue [hunk 2]
```diff
@@ -69,7 +69,7 @@ export default {
   props: {
     /**
      * Array of mimeTypes or a defined preset as String
-     * @see demosplan/DemosPlanCoreBundle/Resources/client/js/lib/FileInfo.js
+     * @see demosplan-ui/src/lib/FileInfo.js
```
Comment (spiess-demos) [inline src/components/core/DpUpload/DpUploadFiles.vue:72]:
```suggestion
     * @see ./src/lib/FileInfo.js
```

File: src/components/core/DpUpload/DpUploadFiles.vue [hunk 3]
```diff
@@ -174,7 +187,7 @@ export default {
      * you can find all available keys here
      * @see https://github.com/transloadit/uppy/blob/master/packages/%40uppy/locales/src/de_DE.js
      * some are already overwritten here
-     * @see demosplan/DemosPlanCoreBundle/Resources/client/js/components/DpUpload/utils/UppyTranslations.js
+     * @see demosplan-ui/src/components/core/DpUpload/utils/UppyTranslations.js
```
Comment (spiess-demos) [inline src/components/core/DpUpload/DpUploadFiles.vue:190]:
```suggestion
     * @see ./src/components/core/DpUpload/utils/UppyTranslations.js
```

File: src/components/core/DpUpload/DpUploadFiles.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import DpUploadFiles from './DpUploadFiles'
+
+<Meta
+    title="Components/UploadFiles"
+    component={DpUploadFiles}
+/>
+
+# UploadFiles
+
+The UploadFiles component can be upload a single file or multiple files.
```
Comment (spiess-demos) [inline src/components/core/DpUpload/DpUploadFiles.stories.mdx:11]:
```suggestion
The UploadFiles component enables users to upload files by dropping them onto a designated area or by triggering the process through a button activation.
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: src/components/core/DpUpload/DpUploadFiles.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+/>
+
+# UploadFiles
+
+The UploadFiles component enables users to upload files by dropping them onto a designated area or by triggering the process through a button activation.
+
+## Custom usage
+
+```html
+<dp-upload-files
```
Comment (spiess-demos) [inline src/components/core/DpUpload/DpUploadFiles.stories.mdx:16]: I'd suggest to only add attrs here which are required.

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: src/components/core/DpUpload/DpUploadFiles.stories.mdx (hunk 5 above)
Comment (ahmad-demos) [reply src/components/core/DpUpload/DpUploadFiles.stories.mdx:16]: Done in https://github.com/demos-europe/demosplan-ui/pull/518/commits/9ec9e27c8b9f8408f9ff481f42a532bc858117bf

Comment (ahmad-demos) [reply src/components/core/DpUpload/DpUploadFiles.stories.mdx:16]: https://github.com/demos-europe/demosplan-ui/pull/518/commits/071b8c4f9a8e0498d8f3f290dcdc209075f19ff0

**Pull request merged**